docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -173,12 +220,12 @@ the action:
     xtrabackup does not copy the keyring file into the backup directory. To prepare the backup, you must copy the keyring file manually.
 
 
-### Prepare the backup with the `keyring_file` component
+### Prepare the backup with the keyring_file component
 
-xtrabackup reads the keyring_file component configuration
+xtrabackup reads the `keyring_file` component configuration
 from `xtrabackup_component_keyring_file.cnf`. You must specify the
 keyring_file data path if the `keyring-file-data` is not located in the
```

Comment:
here too, we use keyring_file_data for component version. It should be --component-keyring-config